tech-far-hub/src/pages/template-listing.tsx

```diff
@@ -115,6 +123,13 @@ export const query = graphql`
           promo_description
           link
         }
+        parent {
+          ... on File {
```

Comment:
are these Fragments? I forgot about that!

Reply:
I wish I could DRY up my graphql more, but these are working pretty well.